### DISCUSSION TITLE
GET /points/{point}/stations deprecated - What is the replacement?

### DISCUSSION THREAD

**Discussion by Styx13** [weather-gov/api#562] [Q&A] [answered]
Hello,

I see in the documentation that `/points/{point}/stations` is deprecated.
It seems to still work, however I wonder for how long ?

And what is the alternate method to find out the closest station based on latitude/longitude ?

Thank you.

**Comment by jah1007** [comment, marked as answer]
You’ll want to go to points/{point} and gather the URL from the observationStations parameter. The output should be listed in order of distance from the point.
